From e4983bad21c2d188d1fb7d7e4cd4d58df1df74ae Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Mon, 14 Aug 2017 10:02:13 +0200
Subject: [PATCH] RHEL 7: v2v: disable unconfig of manually installed VMware
 tools (RHBZ#1477905)

It looks like they may require connection to the VMware servers, which
is not always available during conversion.
---
 v2v/convert_linux.ml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/v2v/convert_linux.ml b/v2v/convert_linux.ml
index 4a7447fe1..7588fbf8e 100644
--- a/v2v/convert_linux.ml
+++ b/v2v/convert_linux.ml
@@ -352,6 +352,7 @@ let convert (g : G.guestfs) inspect source output rcaps =
     let remove = !remove in
     Linux.remove g inspect remove;
 
+(*
     (* VMware Tools may have been installed from a tarball, so the
      * above code won't remove it.  Look for the uninstall tool and run
      * if present.
@@ -373,6 +374,8 @@ let convert (g : G.guestfs) inspect source output rcaps =
           warning (f_"VMware tools was detected, but uninstallation failed.  The error message was: %s (ignored)")
             msg
     )
+*)
+    ()
 
   and unconfigure_citrix () =
     let pkgs =
-- 
2.20.1

